catkit/emulators/iris_ao_controller.py

```diff
@@ -0,0 +1,69 @@
+import abc
+
+import catkit.hardware.iris_ao.util
+
+class IrisAOEmulator(abc.ABC):
+    """ Emulates subprocess for particular use by catkit.hardware.iris_ao.iris_ao_controller.IrisAoDmController. """
+
+    PIPE = "PIPE"
+    CREATE_NEW_PROCESS_GROUP = "CREATE_NEW_PROCESS_GROUP"
```

Comment:
These could be ``None`` for all we care.

Reply:
Ok. Yeah let's make it `None` so that we don't suggest we're actually doing anything with this.